src/main/java/Model/Product.java

```diff
@@ -0,0 +1,13 @@
+package Model;
+public class Product {
+    private String name;
+    private String description;
+    private Float value;
+    private Boolean isOnSale = false;
```

Comment:
Initialize this boolean in the constructor

Reply:
Done